docs/source/tutorials/qmd/07-error-correction.qmd

```diff
@@ -0,0 +1,623 @@
+---
+engine: knitr
+---
+
+```{python}
+#| label: setup
+#| include: false
+
+import liesel.goose as gs
+import pandas as pd
+
+gs.Summary.__repr__ = gs.Summary._repr_html_
+gs.Summary._repr_markdown_ = gs.Summary._repr_html_
+pd.options.display.float_format = "{:.3f}".format
+pd.options.display.html.border = 0
+```
+
+# Bayesian Measurement Error Correction
+
+In this tutorial, we implement a flexible distributional regression model
+with Bayesian measurement error correction in Liesel. The model should capture the
+(potentially) non-linear effect of a continuous covariate $x$, which is
+assumed to be suspect of a measurement error.
+
+## Measurement Error
+
+Assuming a total number of $M$ replicates of the covariate $x$ affected by
+measurement error, we can write the measurement error model as
+
+$$
+\tilde{x}_i^{(m)} = x_i + u_i^{(m)}, \quad m = 1, \ldots, M.
+$$
+
+The measurement error is assumed to follow a multivariate Gaussian model,
+such that
+
+$$
+\mathbf{u}_i \sim N_M(\mathbf{0}, \mathbf{\Sigma}_{u,i}).
+$$
+
+The fundamental concept behind Bayesian measurement error correction is
+to treat the true, unknown covariate values $x_i$ as additional latent variables.
+These values are then imputed using MCMC simulations while simultaneously
+estimating all other parameters of the model. To achieve this, a prior distribution
+is assigned to $x_i$. Additional flexibility is achieved by adding hyperpriors with
```

Comment:
Quite the deep hierarchy. Does tau^2_mu also receive a prior? Or is that one fixed?